ONSdigital/census-rm-acceptance-tests-2021
# RMC-134 Fix fulfilment request message to not error in action scheduler

# Motivation and Context
The action scheduler is now consuming these message to distribute ad hoc action requests so the message needs to have correct data.

# What has changed
* Bring fulfilment request message up to date

# How to test?
Run with the linked branches, action scheduler should not start erroring on the fulfilment request scenario.

# Links
https://github.com/ONSdigital/census-rm-action-scheduler/pull/25
https://github.com/ONSdigital/census-rm-print-file-service/pull/8
https://trello.com/c/ASmhCdSE/726-rmc-134b1-implement-pq-requested-event-adhoc-fulfillment-requests-for-print-files-in-actions-scheduler-part-2-8
